ui/src/router/index.js

```diff
@@ -16,23 +16,23 @@ const routes = [
     // route level code-splitting
     // this generates a separate chunk (about.[hash].js) for this route
```

Comment:
Oh there's another reference to "about". Can you update it to say (query.[hash].js)?

Reply:
Oh, good catch!!! Thanks, I will address it right now.